src/main/target/NAZE/target.h

```diff
@@ -74,36 +82,66 @@
 #define USE_FLASH_M25P16
 
 #define GYRO
+#ifdef NAZE_GENERIC
 #define USE_GYRO_MPU3050
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev5)
 #define USE_GYRO_MPU6050
+#endif
+#if defined(NAZE_GENERIC) || defined(NAZE_rev6)
 #define USE_GYRO_MPU6500
 #define USE_GYRO_SPI_MPU6500
+#endif
+
 
+#ifdef NAZE_GENERIC
```

Comment:
These conditionals should not be in here. Defines that are not used are simply ignored, making them conditional just clutters up the code. If anything, move the `GYRO_X_ALIGN` defines up next to the `USE_GYRO_X` defines.